cluster/src/test/java/org/apache/iotdb/cluster/integration/IoTDBMetadataFetchRemoteIT.java

```diff
@@ -29,7 +29,6 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-<<<<<<< HEAD
 //public class IoTDBMetadataFetchRemoteIT extends IoTDBMetadataFetchAbstract{
```

Comment:
you should add an empty class instead of remove whole class

Reply:
fixed